gems/activesupport/6.0/activesupport.rbs

```diff
@@ -501,7 +501,7 @@ class Array[unchecked out Elem]
   #
   # The differences with <tt>Kernel#Array</tt> explained above
   # apply to the rest of <tt>object</tt>s.
-  def self.wrap: (nil) -> []
+  def self.wrap: (nil) -> Array[untyped]
                | [T] (_ToAry[T] array_like) -> Array[T]
                | [T] (T ele) -> [T]
```

Comment:
Should we also change `[T]` to `Array[T]`? 

It seems that this method is not intended to return a tuple, but rather it just happens to result in an array with one element. Given the name `Array.wrap`, I think it makes more sense for it to return an `Array`, don't you?
```suggestion
               | [T] (T ele) -> Array[T]
```

Reply:
Reasonable. I'm okay to change it.

Reply:
changed.